lib/inline_svg.rb

```diff
@@ -51,7 +59,6 @@ def asset_finder=(finder)
                         # See: https://github.com/jamesmartin/inline_svg/issues/25
                         InlineSvg::StaticAssetFinder
                       end
-      asset_finder
```

Comment:
Return values [are ignored](https://docs.ruby-lang.org/en/2.7.0/syntax/methods_rdoc.html#label-Return+Values) in setter methods.